server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuthenticateCountyAdministrator.java

```diff
@@ -54,15 +52,10 @@ public String endpointName() {
    */
   @Override
   public String endpoint(final Request the_request, final Response the_response) {
-    int status = HttpStatus.OK_200;
-    String result = "Authenticated";
-    
+    ok(the_response, "Authenticated");
```

Comment:
I see that it does set the status, but I think that's OK as long as the status is correct at the end of the endpoint handler... it's not clear to me how Spark deals with it if one streams the response (by circumventing its cache), though, as we do in some cases.